hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/client/functional/SparkClientFunctionalTestSuite.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.hudi.client.functional;
+
+import org.junit.platform.runner.JUnitPlatform;
+import org.junit.platform.suite.api.IncludeTags;
+import org.junit.platform.suite.api.SelectPackages;
+import org.junit.runner.RunWith;
+
+@RunWith(JUnitPlatform.class)
+@SelectPackages("org.apache.hudi.client")
+@IncludeTags("functional")
+public class SparkClientFunctionalTestSuite {
```

Comment:
is this needed?

Reply:
my understanding is yes. thats how any test class thats tagged with "functional" is getting picked up as functional tests.
@xushiyan : Can you confirm.

Reply:
Yes this looks good. `*FunctionalTestSuite.java` is the entrypoint for functional tests in each module. Defined in pom.xml functional test profile

Reply:
thanks. @xushiyan. quick question. If we have java and scala test directory for a module. do we need two FunctionalTestsuite (one for java and one for scala) or having just one in java would suffice?

Reply:
i think one should suffice with `@SelectPackages("")` handles multiple package paths. but please give a try i haven't done it myself :)